src/multiplatform-apps.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Multi-platform app development
+short-title: Multi-platform apps
```

Comment:
Do we really need _development_ in the title? I think it's good to match the sidenav unless there's a compelling reason not to.

```suggestion
title: Multi-platform apps
```